src/main/editDelete/liveProcess.ts

```diff
@@ -0,0 +1,42 @@
+import { Transcription, Word } from 'sharedTypes';
+
+/**
+ * calculateTime calculates the outputStartTimes of a word based on the
+ * current state of all words in a Transcript. It takes into account edits/deletes
+ * @param word word in Transcript
+ * @param i the current index position in the words array
+ * @param words an array of words from Transcript
+ * @returns the input word with an updated outputStartTime
+ *
+ */
+
+const calculateTime = (word: Word, i: number, words: Word[]): Word => {
```

Comment:
Great stuff, would be good to add some comments in this function since the logic is quite complex

Reply:
Comments have been added to calculateTime